Framework/Crystal/inc/MantidCrystal/IntegratePeakTimeSlices.h

```diff
@@ -203,7 +204,7 @@ class DataModeHandler {
   }
 };
 
-class MANTID_CRYSTAL_DLL IntegratePeakTimeSlices : public Mantid::API::Algorithm {
+class MANTID_CRYSTAL_DLL IntegratePeakTimeSlices : public Mantid::API::Algorithm, public API::DeprecatedAlgorithm {
```

Comment:
I don't think you need to have both "public" in there
```suggestion
class MANTID_CRYSTAL_DLL IntegratePeakTimeSlices : public Mantid::API::Algorithm, API::DeprecatedAlgorithm {
```

Reply:
That's correct. Update the code.